packages/fiori-elements-writer/test/common.ts

```diff
@@ -136,11 +146,15 @@ export const projectChecks = async (
             console.log('stderr:', npmResult.stderr);
 
             // run checks on the project
-            // Check TS Types
             if (config.appOptions?.typescript) {
+                // Check TS Types
                 npmResult = await exec(`${npm} run ts-typecheck`, { cwd: rootPath });
                 console.log('stdout:', npmResult.stdout);
                 console.log('stderr:', npmResult.stderr);
+                // Check Eslint
```

Comment:
Just to be sure I understand this test: if an eslint error is found in the generated project then an error is thrown here and the test fails OR is it just pushing things to the console for debugging purposes without testing anything?

Reply:
This logs it to the console for debugging. 
I believe it is the return code of the eslint command that fails the test.

Reply:
So the test fails if there is an eslint error?